### PR TITLE
Allow multiple catalogs in raco pkg install and friends

### PR DESCRIPTION
**Pull request by sorawee**
CC: @mflatt @jackfirth

### CODE REVIEW

File: pkgs/racket-doc/pkg/scribblings/pkg.scrbl [hunk 1]
```diff
@@ -544,8 +544,8 @@ sub-commands.
  @item{@Flag{u} or @DFlag{user} --- Shorthand for @exec{--scope user}.}
  @item{@DFlag{scope-dir} @nonterm{dir} --- Select @nonterm{dir} as the @tech{package scope}.}
  
- @item{@DFlag{catalog} @nonterm{catalog} --- Uses @nonterm{catalog} instead of of the currently configured 
-       @tech{package catalogs}.}
+ @item{@DFlag{catalog} @nonterm{catalog} --- Uses @nonterm{catalog}s instead of of the currently configured
+       @tech{package catalogs}. This flag can be provided multiple times.}
```
[the remark this answers is not in the document]
Comment (sorawee) [reply pkgs/racket-doc/pkg/scribblings/pkg.scrbl]: Should I bump the Racket version up as well?

[the remark this answers is not in the document]
Comment (mflatt) [reply pkgs/racket-doc/pkg/scribblings/pkg.scrbl]: That would be ok, but I don't consider it necessary.

**Comment by mflatt** [comment]
Looks ok to me. Ready to merge?

**Comment by sorawee** [comment]
Yes, feel free to merge :)

**Comment by sorawee** [comment]
Thanks!

**Pull request merged**